frontend/app/src/services/utils.ts

```diff
@@ -105,7 +105,7 @@ export function validWithSessionAndExternalService(status: number): boolean {
  * @return The validity of the status code
  */
 export function validStatus(status: number): boolean {
-  return isValid([200, 400, 409], status);
+  return isValid([200, 400, 404, 409], status);
```

Comment:
The problem with this approach is that 404 is now a valid status code in a number of APIs were it should not be a valid status.

This means that instead of that it will try to parse this response instead of going directly to the error.

If we don't have a validator that has the combination of these statuses, create a new one instead. And check the rest documentation to see that we follow the same pattern as the one mentioned in the response statuses.

Reply:
If the codes are exactly the same the sure, it won't be a problem. 
The problem usually is if a code that is considered valid is not actually valid for the endpoint.